packages/react-component-playground/src/components/display-screen.less

```diff
@@ -5,9 +5,9 @@
 @twinkling-texture: url('../static/twinkling.png');
```

Comment:
Might as well get rid of `twinkling-texture` and `clouds-texture` variables and the corresponding files.